README.md

```diff
@@ -130,6 +181,15 @@ See `dsync --help` for more information.
 
 Feel free to open tickets for support or feature requests.
 
+## Migrating from v0 to v1
+* For those that use plural table names: English-based pluralization is no longer handled. If you had a table named `todos`, dsync will not generate a struct named `Todo` anymore, but instead, `struct Todos` will be generated. It might be easier to rename your tables entirely to singular case.
```

Comment:
```suggestion
## Migrating from v0 to v1

* For those that use plural table names: English-based pluralization is no longer handled. If you had a table named `todos`, dsync will not generate a struct named `Todo` anymore, but instead, `struct Todos` will be generated. It might be easier to rename your tables entirely to singular case.
```